ui/src/components/harvest-games/HarvestGameInstructions.js

```diff
@@ -0,0 +1,357 @@
+                Min player required
+              </Heading>
+            </Box>
+            <Box
+              borderColor="rgba(8, 86, 131, 1)"
+              borderWidth="7px"
+              borderRadius="10px"
+              fontWeight={500}
+              textColor="rgba(151, 149, 149, 1)"
+              fontSize="2vw"
+              w="30%"
+              h={{ base: '17em', sm: '14em', md: '10em', lg: '7em' }}
+              marginX="0"
+              textAlign="center"
+              paddingY={{ base: '2', md: '5', lg: '4' }}
+            >
+              <Heading
+                textColor="rgba(1, 43, 85, 1)"
+                fontWeight={500}
+                fontSize={{ base: '3xl', md: '5xl' }}
+              >
+                {gameType[selected].points}
+              </Heading>
+              <Heading
+                fontSize={{ base: 'sm' }}
+                textColor="rgba(61, 61, 61, 1)"
+              >
+                Points
+              </Heading>
+              <Heading fontSize={{ base: '2vw', md: '1.5vw' }}>
+                Total points for all rounds
+              </Heading>
+            </Box>
+          </HStack>
+          <VStack alignItems="start" marginX="0" w="100%" paddingBottom="10">
+            <Box
+              display={gameType[selected].id === 'game_2' ? 'block' : 'none '}
+              borderColor="rgba(8, 86, 131, 1)"
+              borderWidth="7px"
+              borderRadius="10px"
+              fontWeight={500}
+              fontSize={{ base: '5vw', md: '2.5vw' }}
+              textColor="rgba(61, 61, 61, 1)"
+              w="100%"
+              marginX="0"
+              marginY="5"
+              textAlign="start"
+              paddingY="5"
+              paddingX="9"
+            >
+              Materials :
+              <VStack justifyContent="space-evenly" alignItems="start">
+                {gameType[selected].id === 'game_2' &&
+                  gameType[selected].materials.length > 0 &&
+                  gameType[selected].materials.map((e, i) => {
+                    return (
+                      <Fragment key={i}>
+                        <Text
+                          color="rgba(1, 43, 85, 1)"
+                          fontWeight={400}
+                          textDecoration="none"
+                          fontSize={{ base: '4vw', md: '2vw' }}
+                        >
+                          {e}
+                        </Text>
+                      </Fragment>
+                    );
+                  })}
+              </VStack>
+            </Box>
+            <Heading paddingX="2">Instructions :</Heading>
+            <Grid
+              w="100%"
+              mt="12"
+              mb="12"
+              templateColumns="repeat(1, 1fr)"
+              gap="2"
+            >
+              {gameType[selected].imageUrl.length > 0 &&
+                gameType[selected].imageUrl.map((url, i) => (
+                  <Box
```

Comment:
lol why are you using a whole box and setting the background image instead of just using an `Image` component?

Reply:
It was easier to control the height haha.